code/modules/organs/limbs.dm

```diff
@@ -83,6 +86,7 @@
 		RegisterSignal(owner, COMSIG_PARENT_QDELETING, .proc/clean_owner)
 	soft_armor = getArmor()
 	hard_armor = getArmor()
+	max_damage += DELIMB_BUFFER
```

Comment:
no do it in code manually. 

There is no reason to do something at runtime if you can do it in compilation

Reply:
Did exactly that.